runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingPlugin.java

```diff
@@ -139,6 +143,7 @@ protected void startUp() throws Exception
 
 		eventBus.register(compostTracker);
 		eventBus.register(paymentTracker);
+		eventBus.register(geomancyManager);
```

Comment:
missing matching unregister

Reply:
addressed in 4325d498daf8ff7560b56fe22023e16fbb5e1e0b